### DISCUSSION TITLE
Github integration bug when pasting link with line number selection

### DISCUSSION THREAD

**Discussion by netpoe** [GitbookIO/community#346] [3. Git Sync]
Having installed the github integration, when pasting a github permalink that is meant to render a line range, will render empty code blocks in gitbook:

<img width="1512" alt="Screenshot 2023-09-20 at 12 24 40" src="https://github.com/GitbookIO/community/assets/4053518/62473cb4-6d7e-48d4-89ff-f1696a7f8177">

Tried with this permalink: 

https://github.com/aufacicenta/pulsemarkets/blob/765bf2249e007bdbd64acd22560bc831858383fa/app/src/context/near/prompt-wars-market-contract/NearPromptWarsMarketContractContextController.tsx#L80-L88

**Comment by netpoe** [comment]
@addisonschultz I'd appreciate your help with this one. 

Is Gitbook open-source? I may try to fix this in a PR, just point me in the right direction.

**Comment by SamyPesse** [comment]
Are you using the GitHub Files integration (https://app.gitbook.com/integrations/github-files) ?

In that case, yes it's open source; it might be a bug in the parser we have in https://github.com/GitbookIO/integrations/blob/main/integrations/github-files/src/github.ts with these regexes https://github.com/GitbookIO/integrations/blob/main/integrations/github-files/src/github.ts#L8-L11

Let me know if you try to fix it with a PR :)
